src/components/MetadataEditor.vue

```diff
@@ -345,7 +357,10 @@
          /* we want to have all nested fields to be present for convenience,
             that's what objectFactory essentially does */
          this.value = merge({}, defaultValue, this.fixEntries(lastValue));
+         this.applyDefaultDatabases(this.schema.properties.metaspace_options.properties.Metabolite_Database.items);
```

Comment:
Found a bug here - `this.schema.properties.metaspace_options.properties.Metabolite_Database.items` isn't always an array. Will fix.